.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,27 +29,27 @@ APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps
 To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.
 
 1. [ ] stable
-2. [ ] giftCard
-3. [ ] category
-4. [ ] collection
-5. [ ] attribute
-6. [ ] productType
-7. [ ] shipping
-8. [ ] customer
-9. [ ] permissions
+2. [ ] app
+3. [ ] attribute
+4. [ ] category
+5. [ ] collection
+6. [ ] customer
+7. [ ] giftCard
+8. [ ] homePage
+9. [ ] login
 10. [ ] menuNavigation
-11. [ ] pages
-12. [ ] sales
-13. [ ] vouchers
-14. [ ] homePage
-15. [ ] login
-16. [ ] orders
-17. [ ] products
-18. [ ] app
-19. [ ] plugins
-20. [ ] translations
-21. [ ] navigation
+11. [ ] navigation
+12. [ ] orders
+13. [ ] pages
+14. [ ] payments
+15. [ ] permissions
+16. [ ] plugins
+17. [ ] productType
+18. [ ] products
+19. [ ] sales
+20. [ ] shipping
+21. [ ] translations
 22. [ ] variants
-23. [ ] payments
+23. [ ] vouchers
 
-CONTAINERS=1
+CONTAINERS=2
```

Comment:
suggestion: can we increase this value to 5?

Reply:
Yes we can but if you don't select any check box then 45 tests marked as critical will trigger 5 containers which is overkill - rule of thumb here should be to change the container amount if you select any checkbox